tests/ignite/metrics/test_cohen_kappa.py

```diff
@@ -325,3 +325,14 @@ def _test_distrib_xla_nprocs(index):
 def test_distrib_xla_nprocs(xmp_executor):
     n = int(os.environ["NUM_TPU_WORKERS"])
     xmp_executor(_test_distrib_xla_nprocs, args=(), nprocs=n)
+
+
+def test_skip_unrolling():
+    y_pred = torch.randint(0, 2, size=(8, 1))
+    y_true = torch.randint(0, 2, size=(8, 1))
+    ck = CohenKappa(skip_unrolling=True)
+    state = State(output=(y_pred, y_true))
+    engine = MagicMock(state=state)
+    ck.iteration_completed(engine)
+    res = ck.compute()
+    assert isinstance(res, float)
```

Comment:
Finally, I think we do not need tests for each metric as we already tested base class. Let's remove all such tests. We can keep the test in tests/ignite/metrics/test_accuracy.py though